### DISCUSSION TITLE
Who uses Kimai and to what extent? I am looking for customer testimonials

### DISCUSSION THREAD

**Discussion by MartinMaehlmannMS** [kimai/kimai#4622] [General]
Hi guys, I'd like to know who all works with Kimai and in which environment (e.g. company size) you use it and how you work with it (e.g. complete vacation planning, time accounting for customers, etc. pp.)?

I would be happy about active participation!

Cheers
Martin

**Comment by kevinpapst** [comment]
What about sharing first? And why do you want to know that?

**Reply by MartinMaehlmannMS** [reply]
Hello Kevin,

We would like to use Kimai in our company and would like to have experience reports from other users. To be precise, the management would like to know to what extent the software is used by other companies and whether they are satisfied with it or whether there were any problems with a possible changeover from another system - field reports.

I think Kimai is perfect, it's a great solution that can be flexibly adapted at any time.

**Reply by kevinpapst** [reply]
Ok. I wouldn't expect to much feedback here. Users come here to share problems not success stories.
Thousands of companies are using Kimai... and after talking to hundreds of them I can tell you that every industry and company size is different.

**Reply by MartinMaehlmannMS** [reply]
ok, thanks for your answer. Do you think you can get a testimonial from a larger company? That would probably also be good for Kimai. We would be happy to write you a testimonial after the implementation and would also be available as a reference customer.

**Reply by kevinpapst** [reply]
I understand the value of testimonials, but there is already more work, more requests and more customers that I am capable of handling. Kimai is my business life and I am working on it 10+ hours every day.

If you don't believe numbers (history of Kimai > 15 years, which you can easily find out) or stars or the plugin shop with dozens of developers and hosting companies supporting it .. what shall I say 🤷 

In all these years this is the first request of its kind. It new, its special, but I always willing to help, under the terms of paid support.
If your management feels better if something costs money, I don't mind writing invoices 😁

**Comment by empx92** [comment]
We are a Company with about 20 People. We use it to Manage Customers and Projects which is good for most use cases (90% of Time).
The other 10% are really special features which are needed by our customers. I would say there is nearly nothing you can't get running with some plugins. i'm only stuck with 1 or 2 things... But i also have to say that kevin is making a real good work in that way, there are so many things which he already implemented because of a big demand.

Our Workers add their Time and Description with Tablets to our kimai installation which runs in-house on our server. the secretary checks it prints it out and the workers sign it. because this way it is also usable instead of good old paper if it comes to examination, we got this tip from our payroll accounting - but be aware that i use a self setup export template with all the infos in it wee need . for example in our collective agreement there are some rules like meal money which are dependend of the time away from company and distance for example...

**Comment by Sleepful** [comment]
I use it for my 1 person freelancing with multiple clients, it is awesome, I can have different templates for invoices, in different languages, calculates tax, all that... all great.
